test/lightning_web/live/project_live_test.exs

```diff
@@ -4,6 +4,7 @@ defmodule LightningWeb.ProjectLiveTest do
   import Phoenix.LiveViewTest
   import Lightning.ProjectsFixtures
   import Lightning.AccountsFixtures
+  import SweetXml
```

Comment:
I think we should be using Floki instead of `SweetXml`... Check the code base for other uses of `Floki`.